### ISSUE TITLE
[Request] ImageMultiplier for Visual Studio

### ISSUE THREAD

**Issue by De4th69** [IanMercer/imagemultiplier#4]
Hi,
is it somehow possible to get this working in VS?
Or are you planning to put this on NuGet?
Because you AddIn looks amazing.

Greetings

**Comment by IanMercer** [comment]
Yes, I am considering creating a Visual Studio extension for this as soon as I can find the time.

**Comment by De4th69** [comment]
Awesome :+1: Thank you for your work :)

**Comment by valdetero** [comment]
Yes please this would be awesome.

**Comment by jinlook** [comment]
Please add a standalone console project too, so that we can build it by ourselves and use it everywhere. thanks.

**Comment by danim-hub** [comment]
this fork works on visual studio
CliffCawley/imagemultiplier
